examples/lsm303dlh_mag_tiltcompensatedcompass.py

```diff
@@ -0,0 +1,86 @@
+# Tilt Compensated Compass using Accelerometer and Magnetometer
+# SPDX-FileCopyrightText: 2021 Alex Dew
+# SPDX-License-Identifier: MIT
+
+# The equations below are the ones used in code
+# Equations from st manual:
+# "Computing tilt measurement and tilt-compensated eCompass" by Andrea Vitali
+# https://www.st.com/resource/en/design_tip/dm00269987-computing-tilt-measurement-and-tiltcompensated-ecompass-stmicroelectronics.pdf
+
+# for Phi (Roll)
+# Phi = atan2(Gy, Gz)
+# for stability, 'Gz' can be substituted with Gz+Gx*alpha (alpha=0.01-0.05)
+
+# for Theta (Pitch)
+# Gz2 = Gy * Sin( Phi ) + Gz * Cos( Phi )
+# Theta = Atan(-Gx / Gz2)
+
+# for Psi or "Yaw" (The Tilt Compensated Heading)
+# Bx,By,Bz are magnetometer readings
+
+# By2 = Bz * Sin( Phi ) – By * Cos( Phi )
+# Bz2 = By * Sin( Phi ) + Bz * Cos( Phi )
+# Bx3 = Bx * Cos( Theta ) + Bz2 * Sin( Theta )
+# Psi = Atan2(By2 , Bx3) # This is the Yaw (Tilt Compensated Heading)
+###############################################
+
+import board
+import busio
+from digitalio import DigitalInOut
+import re
+import time
+from math import atan, atan2, degrees, pi, cos, sin
+import adafruit_lsm303_accel
+import adafruit_lis2mdl
+import adafruit_lsm303dlh_mag
```

Comment:
We import this but we never used it in the code. Sorry if I am missing something,but maybe this example could be in the LSM303_Accel library?

Reply:
😱 Wow, you're right.  I used this library to get all of the normal compass example stuff going before adding the tilt compensation.  Would it be best in the lis2mdl library? 
What sensor would be compatible for this library?  ...maybe we could have an example for both?

Reply:
It is up to you,  I think is a good example so you decide where library to use.:)

Reply:
Since this example doesn't actually use the `adafruit_lsm303dlh_mag` library I think it should not be included in this repo as-is. The unused import line should be removed, and the example should live in either the `adafruit_lis2mdl` or `adafruit_lsm303_accel` libraries, or maybe both.

Or alternatively I wonder if it would be possible use both magnetometer and accelerometer from the same breakout instead of needing two. If I understand correctly the LSM303 breakouts contain both accelerometer and magnetometer so possibly a single breakout could be used for both?